server/members/method.js

```diff
@@ -0,0 +1,17 @@
+Meteor.methods({
+  editStatus: function(data) {
+    check(data, {
+      user_id: String,
+      status: String,
+      study_group_id: String
+    });
+
+    if (!this.userId) {
```

Comment:
Here, We should also check if user belongs to the SG or not.
```
const actor = Meteor.user()
if (!actor || !Roles.userIsInRole(actor, ['owner','admin', 'moderator', 'member'], data.studyGroupId )) {
   ...
```

Reply:
Throw 403 access error